### PR TITLE
localstore: add tests for engine.MultiSeek

### PR DESCRIPTION
**Pull request by disksing**

### CODE REVIEW

File: store/localstore/boltdb/boltdb_test.go [hunk 1]
```diff
@@ -138,4 +138,14 @@ func (s *testSuite) TestDB(c *C) {
 	c.Assert(err, NotNil)
 	c.Assert(k, IsNil)
 	c.Assert(v, IsNil)
+
+	m := db.MultiSeek([][]byte{[]byte("z"), []byte("a"), []byte("a1")})
```
Comment (ngaut) [inline store/localstore/boltdb/boltdb_test.go]: Extract to a new function.

File: store/localstore/goleveldb/goleveldb_test.go [hunk 2]
```diff
@@ -95,4 +95,14 @@ func (s *testSuite) TestDB(c *C) {
 	c.Assert(err, NotNil)
 	c.Assert(k, IsNil)
 	c.Assert(v, IsNil)
+
+	m := db.MultiSeek([][]byte{[]byte("z"), []byte("a"), []byte("a1")})
```
Comment (ngaut) [inline store/localstore/goleveldb/goleveldb_test.go]: Extract to a new function.

**Comment by ngaut** [comment]
LGTM

**Comment by shenli** [comment]
LGTM

**Pull request merged**